src/components/campaign-wizard/CampaignWizard.jsx

```diff
@@ -67,7 +67,7 @@ class CampaignWizard extends React.Component {
                     `Choose a name for the campaign and define the period for which data entry will be enabled`
                 ),
                 help: i18n.t(
-                    `Give your campaign a name that will make it easy to recognize in an HMIS hierarchy. Suggested format is REACTIVE_VACC_LOCATION_ANTIGEN(S) _MONTH_YEAR\n
+                    `Give your campaign a name that will make it easy to recognize in an HMIS hierarchy. Suggested format is "RVC {LOCATION} - {ANTIGEN(/ANTIGENS)} - {CAMPAIGN PERIOD}. Example: RCV Shabunda - Measles/Cholera - Jan_Mar 2019.\n
```

Comment:
Apparently, we cannot use `_` either, it was breaking the string. Changed to "-"